R/module_filter_manager.R

```diff
@@ -131,7 +131,10 @@ srv_filter_manager <- function(id, slices_global) {
             module_labels,
             simplify = FALSE,
             function(module_label) {
-              available_slices <- slices_global$module_slices_api[[module_label]]$get_available_teal_slices()
+              available_slices_api <- slices_global$module_slices_api[[module_label]]
+              available_slices <- if (!is.null(available_slices_api)) {
```

Comment:
In what circumstances`available_slices_api` can be `NULL`? `srv_module_filter_manager` is initialized outside of `call_once_when`, so it is available when the app starts. 
I think this is remaining from previous commits when everything was in `call_once_when`

Reply:
I thought about keeping it to prevent crashes in the future, but you're right, it's not needed.

Removed!